### PR TITLE
Church Bells

### PR DESCRIPTION
**Pull request by SphiraI**
## About the PR
- Adds a church bell to be used with station chapels
*idea and sprites by Kezu, proto/sounds put together by me.*

## Why / Balance
"if the jani gets their jani light for convenience and rp, the chapel gets a church bell to signal for sermons" -Kezu
RP purposes, and to give something to chap similar to something like Clown's horn. Would let Chaps use it to get passerby's attention and whatnot.

## Media
image

**_There is a .ogg sound added, but I'm not sure how to make it play nice with git, please someone bwoink me with how_**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Sphiral&Kezu
- add: Added Church Bells! You may hear otherworldly chimes near your local chapel soon...

### CODE REVIEW

**Review by TheShuEd — APPROVED** [review]

**Review by TheShuEd — COMMENTED** [review]
At the moment, the bell sound is a rather long soundtrack, which will continue to play as if the bell is being struck, even if you move away from the bell. I would change the logic to the following: create a collection of bell sounds, and play the sound every time someone physically hits the bell.

**Comment by TheShuEd** [comment]
I don't know what others will say, but I think the current realization.... rather illogical.

**Comment by Tayrtahn** [comment]
I agree with Ed and the proposed solution: the sound is way too long and should be broken up into individual rings. You'll also need to add an entry for it to attributions.yml.

**Review by TheShuEd — CHANGES_REQUESTED** [review]
^

**Comment by SphiraI** [comment]
> At the moment, the bell sound is a rather long soundtrack, which will continue to play as if the bell is being struck, even if you move away from the bell. I would change the logic to the following: create a collection of bell sounds, and play the sound every time someone physically hits the bell.

Yeah, the original idea was that the bell was rocking back and forth from a single interaction, and Kezu was even thinking about making an animation for it. But I'll throw together some sounds and see if I can do the collection thing.

File: Resources/Audio/Effects/attributions.yml [hunk 1]
```diff
@@ -93,6 +93,11 @@
   license: Custom
   source: https://gdc.sonniss.com
 
+- files: ["church_bell1.ogg", "church_bell2.ogg", "church_bell3.ogg", "church_bell4.ogg"]
+  copyright: '"Dramatic church bells" in "Church Bell" on mixkit.co. See https://mixkit.co/license/#sfxFree for license. Editted from stereo to mono, and variations pitched.'
+  license: Custom
+  source: https://mixkit.co/free-sound-effects/church-bell/
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/attributions.yml:99]: > You can’t redistribute the Item on its own, as stock, in a tool or template, or with source files.

I'm seeking other opinions but this sounds like we can't include it?

Comment (SphiraI) [reply Resources/Audio/Effects/attributions.yml]: Sound changed

**Pull request merged**